zcash_client_backend/src/data_api/testing/transparent.rs

```diff
@@ -105,6 +105,22 @@ where
         st.wallet().get_transparent_balances(account_id, height_2),
         Ok(h) if h.get(taddr) == Some(&value)
     );
+    // TODO: Find our why this does not work.
+    
+    // let unshielded_balance = st.wallet().get_wallet_summary(0)
```

Comment:
@nuttycom @str4d  I naively thought this was a straightforward test change. But it results that `get_wallet_summary` return `None` instead while given the test's state,  it should have a balance. 

Bug or feature? 😅